src/ansi-c/c_typecheck_type.cpp

```diff
@@ -1265,14 +1281,10 @@ void c_typecheck_baset::typecheck_c_enum_type(typet &type)
 
   // throw in the enum members as 'body'
   irept::subt &body=enum_tag_symbol.type.add(ID_body).get_sub();
-
+  
```

Comment:
Accidental change? (Consider setting your editor to truncate trailing blanks)

Reply:
thanks, will do :)